src/uct/ib/mlx5/ib_mlx5.inl

```diff
@@ -508,8 +508,9 @@ static UCS_F_ALWAYS_INLINE void uct_ib_mlx5_bf_copy_bb(void * restrict dst,
     UCS_WORD_COPY(__m128i, dst, __m128i, src, MLX5_SEND_WQE_BB);
 #elif defined(__ARM_NEON)
     vst4q_u64(dst, vld4q_u64(src));
-#else /* NO SIMD support */
-    UCS_WORD_COPY(uint64_t, dst, uint64_t, src, MLX5_SEND_WQE_BB);
+#else
+    /* Prevent the compiler to replace by memmove() */
+    *(uct_ib_mlx5_wqe_ctrl_seg_t *)dst = *(uct_ib_mlx5_wqe_ctrl_seg_t *)src;
```

Comment:
the previous patch looked cleaner,  wqe may consist of several different segments and `uct_ib_mlx5_wqe_ctrl_seg_t`  is less than 1 BB

Reply:
ah right thanks, this last minute change was bogus, force pushed the right fix since PR is so small.